app/services/synchronization/nowait_lock_service.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'three_scale/patterns/service'
```

Comment:
:thinking: I don't see this acceptable complication for one service that needs to live outside `app/`. Or maybe require this inside `application.rb` or `environment.rb`?

Reply:
I don't know :shrug: Whatever you prefer :grimacing: I just really hated requiring stuff from `app/lib/` from a module in `lib/`

I don't see requiring this patterns thing terrible. But probably if we continue using this pattern more, we can require it globally.

I am also not sure how valuable using this pattern is, but not opposed to it :)

Reply:
Both should be fine IMO. Zeitwerk maybe wouldn't like requiring from `app/` in dev mode but running rake doesn't invoke reloading classes so should be fine both ways.

wrt usefulness - it is useful if we know how each service works and not read the code to see how to call it. Presently we have far more custom services though.